be/src/runtime/mem_tracker.cpp

```diff
@@ -17,11 +17,9 @@
 
 #include "runtime/mem_tracker.h"
 
-#include <cstdint>
-
-#include <boost/algorithm/string/join.hpp>
 #include <boost/algorithm/string.hpp>
-
+#include <boost/algorithm/string/join.hpp>
```

Comment:
We should avoid use boost libs, we are working to remove all boost lib

Reply:
OK

Reply:
Replace with "util/string_util.h"